platform-docs/docs/intro.md

```diff
@@ -59,10 +59,10 @@ registerCoreBlocks();
 function Editor() {
   const [blocks, setBlocks] = useState([]);
   return (
-    {/*
-        The BlockEditorProvider is the wrapper of the block editor's state.
-        All the UI elements of the block editor need to be rendered within this provider.
-      */}
+		/*
+			The BlockEditorProvider is the wrapper of the block editor's state.
+			All the UI elements of the block editor need to be rendered within this provider.
+		*/
```

Comment:
The spacing seems wrong or at least not consistent with the rest of the code block.

Reply:
ok, I tried preserving the same spacing in the example as there was before (sans the `{}`). 

Side note: would be great if we could format the examples using `prettier` to avoid this in the future. I think that we can use a standard `prettier` config for the `platform-docs` that is separate from the Gutenberg config`. Just adding a `prettier: {}` in the package.json of the platform-docs should be sufficient.

What do you think?

Reply:
Sounds good to me 👍

Reply:
Are you sure you made the right change to the formatting though, it seems to be using tabs for me in the GitHub diff while the rest of the example is using spaces. I'm approving the PR anyway, but maybe it's not entirely fixed.

Reply:
Oh, you're right, I'm not sure why the spaces got converted to tabs.

I've updated it now.

Reply:
Hmm, I tried creating a new PR with improved formatting but it's a bit tricky. 

The default in GB repo is tabs. But formatting the markdown docs in `platform-docs` using the default results in weird indentation.  

I tried configuring prettier only for `platform-docs` to use spaces but that conflicts with some GB eslint rules 
<img width="795" alt="Screenshot 2024-03-21 at 12 37 50" src="https://github.com/WordPress/gutenberg/assets/5417266/7657ccce-82a0-4342-850e-8df1d80aab53">

I tried configuring overrides in https://github.com/WordPress/gutenberg/blob/d30cdd0fab1d1875317896ea4d2ef385eaa957cc/platform-docs/.eslintrc.js to overcome that but it didn't seem to work. 

I might give it another shot in the future but I think I won't spend more time on it right now.